src/app.controller.ts

```diff
@@ -123,7 +130,8 @@ export const updateAccountConditionExpiryDateHandler = async (req: FastifyReques
   loggerService.log('Start - Handle update condition for account request');
   const expiryDate = (req.body as { xprtnDtTm?: string })?.xprtnDtTm;
   try {
-    const { code, message } = await handleUpdateExpiryDateForConditionsOfAccount(req.query as ConditionRequest, expiryDate);
+    const { tenantId } = req as TenantAwareRequest;
```

Comment:
Requires a change in the params of the handler as well.
```suggestion
    const { tenantId } = req;
```